crates/libcontainer/src/container/builder_impl.rs

```diff
@@ -130,22 +123,31 @@ impl<'a> ContainerBuilderImpl<'a> {
             prctl::set_dumpable(false).unwrap();
         }
 
+        // Need to create the notify socket before we pivot root, since the unix
+        // domain socket used here is outside of the rootfs of container. During
+        // exec, need to create the socket before we enter into existing mount
+        // namespace. We also need to create to socket before entering into the
+        // user namespace in the case that the path is located in paths only
+        // root can access.
+        let notify_listener =
```

Comment:
very minor nitpick, but we can directly name this `notify_socket` , so we don't have to specify when building ContainerArgs below

Reply:
Yes

Reply:
b220267